src/utils/number.ts

```diff
@@ -15,11 +20,23 @@ export const formatNumberShort = (num: number): string => {
   }
 
   let formattedValue = value.toFixed(1)
+
+  if (Number(formattedValue) >= 1000) {
+    const tempNumber: string[] = []
+    ;[...formattedValue].forEach((value, idx) => {
```

Comment:
Is this possible to avoid this `;` here? This looks a little bit hacky

Reply:
Unfortunately this comes from prettier...

Reply:
hmm, we could just do: 
```ts
    const formattedValueAsArray = [...formattedValue]
    formattedValueAsArray.forEach((value, idx) => {
```
right?